bioimageio/spec/shared/_resolve_source.py

```diff
@@ -31,6 +31,11 @@
 from .raw_nodes import URI
 
 
+class DownloadCancelled(Exception):
```

Comment:
as this is a private model, maybe we should move `DownloadCancelled` to `common`?

Reply:
We can move later then when needed. I didn't see that it was available in shared.